pentaho-karaf-features/src/main/resources/pentaho-karaf-features-standard.xml

```diff
@@ -255,5 +256,8 @@
         <!-- <feature version="${cxf.karaf.version}">cxf-features-clustering</feature> -->
         <bundle>mvn:commons-collections/commons-collections/${commons.collections.version}</bundle>
     </feature>
-
+    <feature name="pentaho-i18n" version="${project.version}">
+      <bundle>mvn:pentaho/pentaho-i18n-bundle/${dependency.pentaho-osgi-bundles.revision}</bundle>
+      <bundle>mvn:pentaho/pentaho-i18n-webservice-bundle/${dependency.pentaho-osgi-bundles.revision}</bundle>
```

Comment:
We need to make the webservice bundle conditional based on the environment. It should only be installed when CXF is present. The following should do it

        <conditional>
      <condition>cxf</condition>
      <bundle>mvn:pentaho/pentaho-i18n-webservice-bundle/${dependency.pentaho-osgi-bundles.revision}</bundle>
    </conditional>